openshift_performance/ci/scripts/conc_jobs.sh

```diff
@@ -1,30 +1,79 @@
 #!/bin/bash
 
-jobs_amount=300
+jobs_amount=(300 400 600 900)
+results_file=conc_jobs_result.out
 
-function create_jobs()
-{
-for i in $(seq 1 $jobs_amount);
-do
+function create_jobs() {
+  for i in $(seq 1 $1); do
     cat ../content/conc_jobs.yaml | sed "s/%JOB_ID%/$i/g" | oc create -f -
-done
+  done
 }
 
-function wait_for_completion()
-{
-  running=`oc get pods | grep -c Completed`
-  while [ $running -lt $jobs_amount ]; do
+function wait_for_completion() {
+  COUNTER=0
+  running=$(oc get pods | grep job- | grep -c Completed)
+  while [ $running -lt $1 ]; do
     sleep 1
-    running=`oc get pods | grep -c Completed`
+    running=$(oc get pods | grep job- | grep -c Completed)
     echo "$running jobs are completed"
+    COUNTER=$((COUNTER + 1))
+    if [ $COUNTER -ge 1200 ]; then
+      echo "$running pods are still not complete after 20 minutes"
+      break
+    fi
+  done
+}
+
+function prepare_project() {
+  oc new-project svt-conc-jobs-$1
+  oc label namespace svt-conc-jobs-$1 test=concurent-jobs
+}
+
+function delete_project() {
+  oc project default
+  oc delete projects -l test=concurent-jobs --wait=false
+  while [ $(oc get projects -l test=concurent-jobs | grep -c svt-conc-jobs) -gt 0 ]; do
+    echo "Waiting for projects to delete"
+    sleep 5
+  done
+}
+
+function_wait_for_termination() {
```

Comment:
;) I think you want to call function `wait_for_termination` not `function_wait_for_termination`

Reply:
Don't know when that happened but fixing